starknet_py/net/client_models.py

```diff
@@ -267,12 +267,17 @@ class BlockTransactionTraces:
 
 
 @dataclass
-class StorageDiff:
-    address: int
+class StorageEntry:
     key: int
     value: int
 
 
+@dataclass
+class StorageDiff:
```

Comment:
Perhaps naming this `StorageDiffItem` like in the rpc spec or a would make more sense? Since it's not a whole storage diff but a part of one.